ibis/tests/all/test_client.py

```diff
@@ -66,3 +74,31 @@ def test_sql(backend, con, sql):
 
     # execute the expression using SQL query
     con.sql(sql).execute()
+
+
+@pytest.mark.xfail_unsupported
+@pytest.mark.xfail_backends([Clickhouse, Csv, Pandas, Impala, PySpark, Spark])
+def test_nullable_input_output(con, backend):
+    # - Clickhouse, CSVClient and Pandas don't have methods
+    #   create_table or drop_table yet.
+    # - Impala, PySpark and Spark non-nullable issues #2138 and #2137
+    sch = ibis.schema(
+        [
+            ('foo', 'int64'),
+            ('bar', ibis.expr.datatypes.int64(nullable=False)),
+            ('baz', 'boolean*'),
+        ]
+    )
+
+    table_name = 'test_nullable_output_{}'.format(backend.name)
```

Comment:
what is the purpose of this test?

Reply:
it checks if the table created with non-nullable field will return correctly this field as non-nullable